### PR TITLE
Handle empty responses gracefully.

### PR DESCRIPTION
**Pull request by samsullivan**
No more `Error`s returned because a `web` search didn't have any results. Also, `counts()` will now accurately return `[type]: 0` if there are no results.

### CODE REVIEW

**Review by weiser — COMMENTED** [review]
We should have a test that exercises this functionality.  I find that tests help me understand how a thing is supposed to work.

**Comment by samsullivan** [comment]
> We should have a test that exercises this functionality. I find that tests help me understand how a thing is supposed to work.

Agreed, will do.

File: test/search.coffee [hunk 1]
```diff
@@ -124,3 +124,20 @@ describe 'search', ->
           'date'
         ]
         done()
+
+  describe.only 'empty responses', ->
```
Comment (weiser) [inline test/search.coffee]: remove the `.only`.

**Review by weiser — APPROVED** [review]
remove the `.only` and you're 👍.  Thanks for sticking through on this!

File: test/search.coffee (hunk 1 above)
Comment (samsullivan) [reply test/search.coffee]: Crap, thanks!

**Comment by samsullivan** [comment]
I'll wait for Travis to pass with all the unit tests, then merge and open a narrows PR.

**Comment by ieatfood** [comment]
Looks good, nice tests.

**Pull request merged**